src/components/templates/TalkShowStudio/TalkShowStudio.tsx

```diff
@@ -17,7 +17,7 @@ import {
 } from "hooks/video/agora";
 
 import AppButton from "components/atoms/Button";
-import Player, { VideoPlayerProps } from "./components/Player/Player";
+import { Player, VideoPlayerProps } from "./components/Player/Player";
 import { ControlBar } from "./components/ControlBar";
 import Audience from "./components/Audience/Audience";
 import SettingsSidebar from "./components/SettingsSidebar/SettingsSidebar";
```

Comment:
I can't actually comment on it inline since it wasn't modified in this PR, but a couple of lines down from here: `remotesClient`, `screenClient`, `cameraClient` are incorrectly created in global scope, and thus loaded and held in memory from the first load of the app, regardless of whether the `TalkShowStudio` / agora is even accessed/used at all.

---

> RE: https://github.com/sparkletown/sparkle/pull/1653#issuecomment-871297368 + https://github.com/sparkletown/sparkle/pull/1653#issuecomment-871305231 + https://github.com/sparkletown/sparkle/pull/1653#issuecomment-871330197
> 
> > > we should take the time to properly review/assess the changes here and ensure they won't needlessly add risk to the env/github environment when merged
> > 
> > There does seem to be a lot of... Agora stuff... left in the console now in env//github. And this is just on the register page!
> 
> @dmitrii-zolotukhin @whoaitsaimz This is the issue I called out in #tech on slack yesterday ([ref](https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1625029929020100)). The agora clients are being made in global scope, thus initialised on app load regardless of whether this template is even being accessed. That shouldn’t be happening. They should be done within the appropriate hook/react component only when needed.
> 
> _Originally posted by @0xdevalias in https://github.com/sparkletown/sparkle/pull/1653#issuecomment-871783853_

Reply:
> Because of how the `TalkShowStudio` is creating 3 agora clients in global scope within it’s component, it is creating them + holding them in memory from initial app load, regardless of whether the user is actually using that venue/agora at all. This is something that should be removed ASAP, and could be having impacts on the first-load performance of `env/github`
>
> _Originally posted by @0xdevalias in [`#client-github` on Slack](https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1625029929020100)._

Reply:
@dmitrii-zolotukhin Please follow our standards and add the relevant commit hash to the review comment threads where it was resolved, then mark it resolved.

https://github.com/sparkletown/sparkle/pull/1653/commits/1a95556a6dfecdc953e99c7e694c1f6b703180f9